docs/translating.md

```diff
@@ -58,6 +58,14 @@ You can instead use:
 {% blocktrans %}This string will have {{ value }} inside.{% endblocktrans %}
 ```
 
+## Frontend Translations
+The first step in translating the frontend is to create the translation
+files using `./tools/make-frontend-messages`. This command will create
+translation files under `static/locale`, the location can be changed by
+passing an argument to the command, however make sure that the location is
+publically accessible since these files are loaded through XHR in the
+frontend which will only work with publically accessible resources.
+
```

Comment:
I think the workflow of uploading the `.json` files should be same as Django's `.po` files. The way it is implemented right now, we don't merge frontend translation files with backend translation files. If we want to merge the frontend and backend files we will have to convert the JSON files into `.po` files and then convert back to JSON format. There is a utility available which converts between the two formats.

I'll check the workflow of Transifex for better understanding.